resources/com.sap.piper/pipeline/stageDefaults.yml

```diff
@@ -23,6 +23,9 @@ stages:
         filePattern: '**/*.bats'
       karmaExecuteTests:
         filePattern: '**/karma.conf.js'
+      npmExecuteScripts:
+        npmScripts:
+          - 'ci-frontend-unit-test'
```

Comment:
Should rather be as here: https://github.com/SAP/jenkins-library/pull/2357/files#diff-557c743d40b8865c7f78a448443832645379483bb2119da8f56ac1e364256bb4R15 for the same reason as the user has to define the name of the script in the config,

Reply:
Thanks!